### DISCUSSION TITLE
pac package deploy error

### DISCUSSION THREAD

**Discussion by dvanels** [microsoft/powerplatform-build-tools#715] [Q&A]
Hi,

I need to use this command to deploy a package but I receive an error, that the "deploy" command is not in this context

![image](https://github.com/microsoft/powerplatform-build-tools/assets/157122321/350f94fb-cc49-4591-9cb1-8f06d0c8faff)

Any help is apreciated.

**Comment by parvezghumra** [comment]
@dvanels It looks like a new version of the PAC CLI was released 2 days ago, have you tried this latest version?
https://www.nuget.org/packages/Microsoft.PowerApps.CLI/1.30.3

**Reply by dvanels** [reply]
Hi @parvezghumra ,

I didn't know about this. I will try it.

Thanks!

**Reply by dvanels** [reply]
Yesssss, it is working perfectly!

Thanks @parvezghumra
